lego/apps/frontpage/views.py

```diff
@@ -90,6 +93,8 @@ def get_serializer_context():
 
         queryset_poll = Poll.objects.filter(pinned=True).order_by("created_at").last()
 
+        queryset_interestgroups = AbakusGroup.objects.filter(type=GROUP_INTEREST, active=True).order_by("?")
```

Comment:
Agreed :100:  i though it was a bit unnecessary to randomize it all the time. Ill look into randomQuote and how it behaves with SSR:)

Reply:
The intended behavior is for interestgroups to change on hard reload, i.e. same behavior as quotes. SSR seems fine, but my bad code in fronted gives unwanted behavior since I'm just using the first three quotes from state, which means they will be the same whenever all interestgroups are in the state:/   
I'm considering adding a `fetchRandomWithType(n: number, type: string)`-action in frontend + save to state (iether groups.`randomInterestGroupIds` or just `randomGroupIds`, similar to `quote.randomQuoteId`) to solve it. It's way more overhead in frontend and backend, but could be useful any randomized fetching of groups we might need in the future. Thoughts?